Exercises/kokkoskernels/GaussSeidel/Begin/gauss_seidel.cpp

```diff
@@ -81,7 +81,7 @@ int main(int argc, char* argv[])
     //Get approx. 20 entries per row
     //Diagonals are 2x the absolute sum of all other entries.
     Offset nnz = numRows * 20;
-    Matrix A = KokkosKernels::Impl::kk_generate_diagonally_dominant_sparse_matrix<Matrix>(numRows, numRows, nnz, 2, 100, 1.05 * one);
+    Matrix A = KokkosSparse::Impl::kk_generate_diagonally_dominant_sparse_matrix<Matrix>(numRows, numRows, nnz, 2, 100, 1.05 * one);
```

Comment:
Do we rally showcase using things from `Impl::`?

Reply:
This is one of the functions we use to generate random matrices for unit tests. It's used here because there isn't a similar function outside of Impl::. Users ordinarily don't need it because their matrices would come from Tpetra or some other data source.

We could look into moving these out of Impl though.